services/web-app/lib/github.js

```diff
@@ -58,13 +62,39 @@ export const getRemoteMdxData = async (repoParams, mdxPath) => {
 
   const dirPath = response._links.html.split('/').slice(0, -1).join('/')
 
-  return serialize(usageFileSource, {
-    mdxOptions: {
-      remarkPlugins: [remarkGfm, unwrapImages],
-      rehypePlugins: [[rehypeUrls, mdxImgResolver.bind(null, dirPath)]]
-    },
-    parseFrontmatter: true
-  })
+  let serializedContent = null
```

Comment:
MDX modification logic should all be moved out of this file and into the sanitizer plugin.

The github file should only be responsible for retrieving data from github.com. It should not peek at (or modify) its contents in any form.

There likely needs to be a separate module or component whose responsibility is to call the github.js functions to obtain the data, set up a remark instance with all of the plugins, and produce sanitized mdx as output.